buildscripts/02_toolchain

```diff
@@ -49,12 +42,11 @@ if ($C->HOST ne $C->TARGET) {
 
    # Install a standalone Android NDK toolchain.
    must_system (
-      $C->NDK_HOME . "/build/tools/make-standalone-toolchain.sh",
-      "--stl=$STL{$C->TARGET}",
-      "--ndk-dir=" . $C->NDK_HOME,
-      "--toolchain=$TOOLCHAIN{$C->TARGET}-clang",
-      "--install-dir=$toolchain",
-      "--platform=$PLATFORM{$C->TARGET}",
+      $C->NDK_HOME . "/build/tools/make_standalone_toolchain.py",
+      "--arch", $ARCH{$C->TARGET},
+      "--install-dir", $toolchain,
+      "--api", $API{$C->TARGET},
+      "--force",
```

Comment:
I added it because without it, it didn't work, and then it told me to add `--force`. I don't care, it works now, so good.